NEWS.md

```diff
@@ -1,9 +1,15 @@
 # tune 0.0.1.9000
 
+## Breaking Changes
+
+* The arguments to the main tuning/fitting functions (`tune_grid()`, 
+  `tune_bayes()`, etc) have been reordered to better align with parsnip's `fit()`.
```

Comment:
Maybe another sentence or two: "Specifically, rather than having the recipe or formula as the first argument, you now pass the model first. The recipe and formula methods have been soft deprecated and will be removed in version ___"

Reply:
Oooh I think you added the version note right after i added this review comment

Reply:
Added more detail in 8b36a2f707eae569b252406191a4adebf76aa5eb